src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -446,6 +453,9 @@ export const CheckoutPayment = ({
 
   return (
     <CheckoutPageWrapper handleClickBackButton={handleClickBackButton}>
+      {is3dsError && (
+        <ThreeDSErrorModal isVisible onClose={() => setIs3dsError(false)} />
+      )}
```

Comment:
Perhaps it's intended to be implemented as:

```suggestion
      <ThreeDSErrorModal isVisible={is3dsError} onClose={() => setIs3dsError(false)} />
```